README.md

```diff
@@ -27,6 +27,10 @@ See also [Cocorico](https://github.com/coq/coq/wiki) (the Coq wiki),
 and the [Coq FAQ](https://github.com/coq/coq/wiki/The-Coq-FAQ),
 for additional user-contributed documentation.
 
+## Contributing
+
+Guidelines for contributing to Coq in various ways are listed in the [contributor's guide](CONTRIBUTING.md).
+
```

Comment:
Indeed, I don't think I like this change. I'm unsure this makes it easier to find "Contributing", and it moves it away from "Bug reports" and in between two sections that are both about documentation. What I would agree with would be to reduce the number of main sections (for instance by making Changes a sub-section of Documentation, and Bug reports a sub-section of Contributing).

Also, note that now the contributing guide is linked to from the main page of the Coq website https://coq.inria.fr/